apm-agent-core/src/main/java/co/elastic/apm/agent/impl/ElasticApmTracer.java

```diff
@@ -383,20 +384,27 @@ public void endSpan(Span span) {
             span.decrementReferences();
             return;
         }
-        if (!span.isComposite()) {
+        SpanConfiguration spanConfiguration = getConfig(SpanConfiguration.class);
+        if (span.isExit()) {
+            if (span.getDuration() < spanConfiguration.getExitSpanMinDuration().getMicros()) {
```

Comment:
Does this handle the composite span case where the span is less than the config but the composite span is larger?

Reply:
yes, because `duration = start - end` and start/end is the min/max value of all spans that compose the composite span

Reply:
excellent, thanks